libs/platform/platform_linux.cpp

```diff
@@ -264,7 +264,14 @@ time_t Platform::GetFileCreationTime(std::string const & path)
 #if __GLIBC__ >= 2 && __GLIBC_MINOR__ >= 28
   struct statx st;
   if (0 == statx(AT_FDCWD, path.c_str(), 0, STATX_BTIME, &st))
-    return st.stx_btime.tv_sec;
+  {
+    if (st.stx_btime.tv_sec != 0)
+      return st.stx_btime.tv_sec;
+
+    LOG(LWARNING,
```

Comment:
If it is true, that this is regularly happening with Orb, then this should most likely not be a warning, but info or debug.
What should be a warning is the missing error handling, the else branch if `statx` doesn't return 0,
and then also with checking of `errno`.
Furthermore the macro condition on line 264 is also probably incorrect, since it limits to GLIBC, when musl also supports `statx`.

Reply:
- It should be a warning, because it is a bug that leads to the change of behavior, and users should be aware of that. However, the impact is not critical in OM, at least now.
- Agree about error handling, added ERROR logs there (will abort in debug and print error in release)
- Valid point about musl (how popular is it btw?), updated defines.

Reply:
> Valid point about musl (how popular is it btw?

The most popular phone distro uses `musl`.